matrix_sdk/src/models/room.rs

```diff
@@ -104,6 +110,12 @@ pub struct Room {
     pub creator: Option<UserId>,
     /// The map of room members.
     pub members: HashMap<UserId, RoomMember>,
+    /// A queue of messages, holds no more than 10 of the most recent messages.
+    ///
+    /// This is helpful when using a `StateStore` to avoid multiple requests
+    /// to the server for messages.
+    #[serde(with = "super::message::ser_deser")]
```

Comment:
I think we can  leave this as is for now but be aware that I really want this to be optional somehow so it'll change in the future.

Reply:
Would having this behind a feature flag work? My other idea was to only interact with the state store so there could be `StateStore::store/load_messages` although that would create blocks in `matrix-sdk` that would also be feature gated.

Reply:
Feature flag sounds good.